README.md

```diff
@@ -0,0 +1,713 @@
+# node-diacritics-transliterator
+
+> Diacritic transliteration tools using diacritics.io API
+
+# Installation
+
+```console
+$ npm install diacritics-transliterator
+```
+
+# Usage
+
+```js
+var diacritics = require("diacritics-transliterator");
+diacritics.setVersion("v1");
+var de = diacritics.getLanguage("German");
+// {"de":{...}}
+```
+
+# API
+
+## Contents
+
+- [Initializing](#initializing)
+- [Basic functions](#basic-functions)
+  - [diacritics.setVersion(version)](#diacriticssetversionversion)
+  - [diacritics.getVersion()](#diacriticsgetversion)
+  - [diacritics.getLanguage(code)](#diacriticsgetlanguagecode)
+  - [diacritics.getVariant(code)](#diacriticsgetvariantcode)
+  - [diacritics.getAlphabet(code)](#diacriticsgetalphabetcode)
+  - [diacritics.getContinent(code)](#diacriticsgetcontinentcode)
+  - [diacritics.formatUnicode(string)](#diacriticsformatunicodestring)
+- [Data processing functions](#data-processing-functions)
+  - [diacritics.getDiacritics(string)](#diacriticsgetdiacriticsstring)
+  - [diacritics.getBase(array)](#diacriticsgetbasearray)
+  - [diacritics.getDecompose(array)](#diacriticsgetdecomposearray)
+- [Transliteration functions](#transliteration-functions)
+  - [diacritics.transliterate(string [, type][, variant]))](#diacriticstransliteratestring--type-variant)
+  - [diacritics.createRegExp(string [, options])](#diacriticscreateregexpstring--options)
+  - [diacritics.replacePlaceholder(string [, options])](#diacriticsreplaceplaceholderstring--options)
+
+## Initializing
+
+Each API method requires the `diacritics-transliterator` module.
+
+```js
+var diacritics = require("diacritics-transliterator");
+```
+
+## Basic functions
+
+### diacritics.setVersion(version)
+
+Upon initialization, this module is set to the latest major release version of the diacritics API. To change the version, use this function to set the API to the desired _major_ release.
+
+The `version` parameter (`string` or `number`) will accept the following values:
+
+- `v#`, where `v` is the abbreviation for "version" and `1` is the major release value.
```

Comment:
Umm, yes? LOL

<img title=":_ tuzki_dizzy :" src="https://cloud.githubusercontent.com/assets/136959/13203535/61480e08-d880-11e5-9ea8-7fc03c7d7b56.gif">

Moot point if you think the entire entry needs to be removed.